Makefile

```diff
@@ -157,6 +157,11 @@ ifndef ES_PASSWD
 	$(error ES_PASSWD is not set)
 endif
 
+	docker run -e ES_PASSWD="$(ES_PASSWD)" \
+             -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json \
+             -e CLIENT_DOCKER_IMAGE="spacemeshos/$(DOCKER_IMAGE_REPO):$(BRANCH)" \
+             -it go-spacemesh-python pytest -s test_p2p.py$(DELIM)$(TEST) --tc-file=config.yaml --tc-format=yaml
```

Comment:
why do it in a separate file and not a separate folder like hare or sync?

Reply:
It is just easier. I'll move it to new directory